Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -962,6 +942,366 @@ def get_mapping_fields_command(service):
     demisto.results(types_map)
 
 
+def enrich_new_notables(service, enabled_enrichments, incidents, num_enrichment_events):
+    drilldown_status, asset_status, identity_status = False, False, False
+    num_enriched_notables = 0
+    num_inc = 0
+    not_yet_enriched_notables = demisto.getIntegrationContext().get(NOT_YET_ENRICHED_NOTABLES, [])
```

Comment:
why are we using the integration context and not the last run object?

Reply:
we use both of them. we store the data in the integration context and the metadata for the fetch in the last run object.

Reply:
yea but i asked why